docs/index.yml

```
@@ -117,57 +103,67 @@ conceptualContent:
         - url: framework/index.yml
           itemType: overview
           text: .NET Framework (Windows) apps
+        - url: /answers/products/dotnet
+          itemType: get-started
+          text: .NET on Q&A
```

I suspect most folks don't know Q&A is a product name. Consider removing "on".

```suggestion
          text: .NET Q&A
```